datalad/distribution/dataset.py

```diff
@@ -384,16 +391,20 @@ def get_superdataset(self, topmost=False):
             # None was found
             return None
 
-        if realpath(self.path) != self.path:
-            # we had symlinks in the path but sds_path would have not
-            # so let's get "symlinked" version of the superdataset path
-            sds_relpath = relpath(sds_path, realpath(self.path))
-            sds_path = normpath(opj(self.path, sds_relpath))
+        # I think we might have accounted for it with use of normpath there??
```

Comment:
accounted for having symlinks in the self.path somewhere.  Originally it was due to the fact `GitRepo.get_toppath` was resolving symlinks and was also encumbered by having `/leading/subdir/..` without normalization which precluded current implementation from deducing the path without resolving. using `normpath` seemed to mitigate that, and thus this whole buggy post-check was commented out. But I left it in to possibly ease merging in later on  #990